src/basic/fs-util.h

```diff
@@ -23,6 +23,9 @@
 #define PTR_TO_MODE(p) ((mode_t) ((uintptr_t) (p)-1))
 #define MODE_TO_PTR(u) ((void *) ((uintptr_t) (u)+1))
 
+/* We reuse __O_LARGEFILE to tell xopenat() it should do security labeling. */
+#define O_LABEL __O_LARGEFILE
```

Comment:
hmm, what? O_LARGEFILE leaks into our environment everywhere, please don't use it like that. for example fcntl(F_GETFD) returns that. this is a hack. find another way